proto/klyraprotocol/yields/genesis.proto

```
@@ -0,0 +1,7 @@
+syntax = "proto3";
+package klyraprotocol.yields;
```

Buf (1.47.2)

Files with package "klyraprotocol.yields" must be within a directory "klyraprotocol/yields" relative to root but were in directory "proto/klyraprotocol/yields".

(PACKAGE_DIRECTORY_MATCH)